src/handbook/company/board.md

```diff
@@ -39,3 +39,11 @@ FlowFuse investors get a monthly update sent to them, which contains the KPI's w
 information from the CEO on company performance.
 
 Major investors have access to more data on company performance through Carta.
+
+The company must deliver the following deliverables to all Major Investors:
+* Unaudited financial statements within 120 days after each fiscal year.
+* Unaudited financials within 45 days after each fiscal quarter.
+* Cap table if requested, no more than once a quarter.
```

Comment:
They've got access to this through Carta. They can, and must IMO, self service this.

Reply:
@emilieschario Let the investors let us know, and move forward with carta downloads.

```suggestion
* Cap table is available through Carta
```